tests/tests_pytorch/trainer/dynamic_args/test_multiple_eval_dataloaders.py

```diff
@@ -104,62 +104,3 @@ def val_dataloader(self):
     )
 
     trainer.fit(model)
-
-
-def test_multiple_optimizers_multiple_dataloaders(tmpdir):
```

Comment:
Is this test meaningful? If I would convert this to manual optimization, what exactly would there be to assert?

Reply:
I agree with removing it